### ISSUE TITLE
d

### ISSUE THREAD

**Issue by Ilyabasharov** [Ilyabasharov/instance_tracker#4]
repetitions = 300
timings = torch.zeros((repetitions, 1))
start = torch.cuda.Event(enable_timing=True)
end = torch.cuda.Event(enable_timing=True)
#GPU-WARM-UP
with torch.no_grad():
    for _ in range(10):
        _ = model(dummy_input)

with torch.no_grad():
    for rep in range(repetitions):
        start = torch.cuda.Event(enable_timing=True)
        end = torch.cuda.Event(enable_timing=True)
        start.record()
        _ = model(dummy_input)
        end.record()

        # Waits for everything to finish running
        torch.cuda.synchronize()

        curr_time = start.elapsed_time(end)
        timings[rep] = curr_time

mean_syn = torch.sum(timings) / repetitions
std_syn = torch.std(timings, dim=-1)
print(mean_syn)
